src/Utils.ps1

```diff
@@ -84,7 +84,8 @@ function Add-PoshGitToProfile([switch]$AllHosts, [switch]$Force, [switch]$WhatIf
     $profilePath = if ($AllHosts) { $PROFILE.CurrentUserAllHosts } else { $PROFILE.CurrentUserCurrentHost }
 
     # Under test, we override some variables using $args as a backdoor.
-    if ($args.Count -gt 0) {
+    # TODO: Can we just turn these into optional parameters with well-defined behavior?

```

Comment:
@rkeithhill this question is for you. Can we just make `$ProfilePath` and real (optional) argument that is documented to override `$AllHosts`?